packages/webapp/src/util/moment.js

```diff
@@ -64,3 +79,14 @@ export const getNotificationCardDate = (date) =>
   moment(date).locale(getLanguageFromLocalStorage()).format('MM/DD/YY');
 
 export const getCurrentDateLong = (date) => moment().format('L');
+
+/**
+ * Converts a date string to an ISO string with local timezone, safe to store and retrive from the database.
+ *
+ * @param {string} dateString - The date string to convert, e.g. the date string from a date picker.
+ * @returns {string} The converted ISO string with local timezone, e.g. "2024-03-12T00:00:00.000-07:00" (= midnight PST on March 12, 2024)
+ */
+export function toLocalISOString(dateString) {
+  const date = moment(dateString);
+  return date.format('YYYY-MM-DD[T00:00:00.000]Z');
+}
```

Comment:
This utility function won't be needed until the `handleSubmit` is implemented in the container ticket, but I wanted to get it ready.

Please let me know if there is some existing functionality like this already; I'm surprised I couldn't find one because this sounded like what we are aiming for as the "ideal database format for dates" based on the discussion in #2962 and its [Jira ticket](https://lite-farm.atlassian.net/browse/LF-3799).

Reply:
Just a quick question about this. 

If I "the farmer" decide to enter receipts in the winter while vacationing in Fiji. And I enter Sept 23, 2023 as the date on the receipt. Will it enter Fiji's timezone or my farms timezone?

Reply:
@Duncan-Brain It's browser time, so midnight Fiji time on that date (assuming that you reset your computer's clock while on your vacation).

Reply:
I would very much like to somehow change our default to be farm timezone haha -- but this is consistent with what we already have 👍